src/components/Conference.tsx

```diff
@@ -404,8 +405,8 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
 
         navigator.mediaDevices.getUserMedia(webcamScreenConstraints)
             .then(stream => {
-                this.localStream = stream;
-                this.gotStream(stream);
+                this.localCamStream = stream;
+                this.setLocalStream(stream);
```

Comment:
this.setLocalStream has not been implemented

Reply:
Good catch. Merge issue.

Reply:
done